client/src/ui-components/NavbarComponentCSS.css

```diff
@@ -0,0 +1,23 @@
+.navbar{
```

Comment:
Put the navbar component and css together in a subfolder `ui-components/NavbarComponent/...`